tests/acceptance/resolve-guid-test.ts

```diff
@@ -52,7 +52,14 @@ module('Acceptance | resolve-guid', hooks => {
 
             await visit(`/${file.id}`);
 
-            routingAssertions(assert, '--file', `/${file.id}`, 'guid-file');
+            routingAssertions(assert, '--file', `/${file.id}`, 'guid-file.index');
+        });
+        test('Metadata', async assert => {
+            const file = server.create('file', { target: server.create('registration') });
+
+            await visit(`/${file.id}/metadata`);
+
+            routingAssertions(assert, '--file', `/${file.id}/metadata`, 'guid-file.metadata');
```

Comment:
Is this supposed to be in this PR?

Reply:
I did not write this code but can update it if you like. Would you like all of lines 55-62 removed?

Reply:
This is here because your [first commit](https://github.com/CenterForOpenScience/ember-osf-web/pull/1791/commits/f8d2ce55453516c0992072979769bb95bb880901), for some reason, was resolving a merge conflict, and it was resolved incorrectly. Everything in that first commit (the two things I mentioned inline code review and the file `app/guid-file/metadata/template.hbs`) should be removed. So yes, please remove all of this.

Remember: if it's in your diff, and you didn't put it there, chances are good that it shouldn't be in the PR.

Reply:
This should now be fixed.